feature-libs/product-configurator/rulebased/components/attribute/types/checkbox/configurator-attribute-checkbox.component.ts

```diff
@@ -100,4 +102,11 @@ export class ConfiguratorAttributeCheckBoxComponent
       isLightedUp: value.selected,
     };
   }
+
+  getAriaLabel(
```

Comment:
There is no need to define this method. 
`getAriaLabelGeneric` could be called directly in HTML tree.
See `createAttributeUiKey` in `configurator-attribute-header.component.html`